nss-day-cohort-60/national-parks-django-api
# Jd photo migration

# Description
Migrated images stored in cloudinary to local server storage. Rewrote sql INSERT statements to insert updated image locations

# Steps to test

1. Fetch the client JD-photo-migrations branch and switch to it
2. Fetch the JD-photo-migration server branch and switch to it
3. Delete local db.sqlite file and migration files
4. In terminal run:  python3 manage.py makemigrations
5. In terminal run:  python3 manage.py migrate
6.  In terminal run:  python3 manage.py loaddata users tokens event_types
--From Browser:
7. Visit /home and verify park photos still display in carousel
8. Visit /parks/# and verify carousel, blog photos, campground photos, wildlife photos render correctly
9. Visit Blogs and verify correct images
10. Visit User Hub and verify carousel, blog images display correctly
11. Check network tab in dev tools and confirm no traffic with cloudinary - images should all come from some variation of localhost:8000/media/photos/{image file name}
